src/bin/cli/mod.rs

```diff
@@ -274,3 +340,46 @@ fn parse_import_map(map: &str) -> Result<(String, PathBuf), String> {
         Err("contains no '='".to_owned())
     }
 }
+
+fn import_map_vec<'de, D>(deserializer: D) -> Result<Option<Vec<(String, PathBuf)>>, D::Error>
+where
+    D: serde::Deserializer<'de>,
+{
+    let vec: Option<Vec<String>> = Option::deserialize(deserializer)?;
+    match vec {
+        Some(values) => {
+            let results: Result<Vec<(String, PathBuf)>, String> = values
+                .into_iter()
+                .map(|value| parse_import_map(value.as_str()))
+                .collect();
+
+            match results {
+                Ok(mapped) => Ok(Some(mapped)),
+                Err(err) => Err(serde::de::Error::custom(err)),
+            }
+        }
+        None => Ok(None),
+    }
+}
+
+fn emit_de<'de, D>(deserializer: D) -> Result<Option<String>, D::Error>
+where
+    D: serde::Deserializer<'de>,
+{
+    let str: Option<String> = Option::deserialize(deserializer)?;
+    match str {
+        Some(value) => {
+            match value.as_str() {
+                "ast-dot"|"cfg"|"llvm-ir"|"llvm-bc" |"object"| "asm" => {
+                    Ok(Some(value))
+                },
+                _ => {Err(serde::de::Error::custom("Invalid option for `emit`. Valid options are: `ast-dot`, `cfg`, `llvm-ir`, `llvm-bc`, `object`, `asm`"))}
+            }
+        }
+        None => Ok(None),
+    }
+}
+
+fn default_true() -> bool {
+    true
+}
```

Comment:
As per this PR https://github.com/vityafx/serde-aux/pull/23, you don't need this function. You could just write `default_true`.

Reply:
for some reason, `bool_true` doesn't work